src/test/java/karuna/karuna_backend/visitor/message/domain/VisitorMessageTestConfiguration.java

```diff
@@ -1,13 +1,14 @@
 package karuna.karuna_backend.visitor.message.domain;
 
+import karuna.karuna_backend.TestEmailSender;
 import lombok.AccessLevel;
 import lombok.NoArgsConstructor;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 class VisitorMessageTestConfiguration {
 
     static VisitorMessageService visitorMessageService() {
-        return new VisitorMessageService(new InMemoryVisitorMessageRepository());
+        return new VisitorMessageService(new InMemoryVisitorMessageRepository(), new TestEmailSender());
     }
```

Comment:
Tests are not finished right?

Reply:
why? u cant unit test this code 🗡️  its not needed